cmake/projects/CMakeInSubdir/hunter.cmake

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2015, David Hirvonen
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cacheable)
+include(hunter_source_subdir)
+
+hunter_cacheable(CMakeInSubdir)
+
+# List of versions here...
+hunter_add_version(
+  PACKAGE_NAME
+  CMakeInSubdir
+  VERSION
+  "0.0.1"
+  URL
+  "https://github.com/NeroBurner/cmake_in_subdir/archive/v0.0.1.tar.gz"
+  SHA1
+  5791cd5e7e12c30ba1507f1ff29688aa1b9333f5
+)
+
+# set the sub-directory the projects CMakeLists.txt is in
```

Comment:
Put this extra option in the docs. I just added the doc site for our new fork, so everything will continue to be updated.

Reply:
added documentation. As example it points to zstd project, hope this will hold true in the near future :)